.eslintrc.js

```diff
@@ -87,21 +93,15 @@ module.exports = {
       parserOptions: {
         sourceType: 'module',
       },
-      plugins: ['@typescript-eslint', 'ember-data', 'simple-import-sort', 'import'],
-      extends: ['eslint:recommended', 'plugin:@typescript-eslint/eslint-recommended'],
+      plugins: ['@typescript-eslint', 'ember-data'],
+      extends: ['plugin:@typescript-eslint/eslint-recommended'],
       rules: {
+        'no-restricted-globals': ['off'],
```

Comment:
Question: are we good with *completely* disabling this, or do we want it to be for a specific subset, e.g. `Promise`? (I assume "completely" is fine, but figured I'd double-check!)

Reply:
completely is fine (Promise was the only global we were checking for previously), also we cut to IE11 support in 4 weeks, but also this eslint rule does not appear to have a syntax that allows you to disable only one global, in which case we'd have to manually specify every global we wanted to check except for those we didn't.